modules/andes-core/broker/src/main/java/org/wso2/andes/store/file/FileStoreConnection.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.andes.store.file;
+
+import org.apache.log4j.Logger;
+import org.iq80.leveldb.DB;
+import org.iq80.leveldb.Options;
+import org.wso2.andes.configuration.util.ConfigurationProperties;
+import org.wso2.andes.kernel.AndesException;
+import org.wso2.andes.kernel.DurableStoreConnection;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.iq80.leveldb.impl.Iq80DBFactory.factory;
+
+import static org.wso2.andes.store.file.FileStoreConstants.BLOCK_SIZE;
+import static org.wso2.andes.store.file.FileStoreConstants.CACHE_SIZE;
+import static org.wso2.andes.store.file.FileStoreConstants.MAX_OPEN_FILES;
+import static org.wso2.andes.store.file.FileStoreConstants.PATH;
+import static org.wso2.andes.store.file.FileStoreConstants.WRITE_BUFFER_SIZE;
+
+/**
+ * File connection class. Initiate file based store.
+ */
+public class FileStoreConnection extends DurableStoreConnection {
+    private static final Logger logger = Logger.getLogger(FileStoreConnection.class);
+    private DB brokerStore;
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void initialize(ConfigurationProperties connectionProperties) throws AndesException {
+        super.initialize(connectionProperties);
+
+        try {
+            int mb = 1048576;
```

Comment:
What does this property do? If its a constant value we need to add it as a constant with a meaningful name

Reply:
That value represents the no. of bytes for 1 MB. Added it as a constant.